app/js/lib/foxbox/services/ip-camera.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+import BaseService from './base';
+
+const TYPE = 'ip-camera';
+
+export default class IpCameraService extends BaseService {
+  constructor(props, config) {
+    super(props, config);
+    Object.seal(this);
+  }
+
+  // Public getter only properties.
```

Comment:
It's more for code organisation so that we have a consistent ordering across classes.
If you think that's useless and we're going to keep this ordering, then I can remove them.